notebooks/02- Retail Sales DLT.sql

```diff
@@ -0,0 +1,46 @@
+-- Databricks notebook source
+CREATE INCREMENTAL LIVE TABLE customers
+COMMENT "The customers buying finished products, ingested from /databricks-datasets."
+TBLPROPERTIES ("quality" = "mapping")
+AS SELECT * FROM cloud_files("/databricks-datasets/retail-org/customers/", "csv")
+
+-- COMMAND ----------
+
+CREATE INCREMENTAL LIVE TABLE sales_orders_raw
+COMMENT "The raw sales orders, ingested from /databricks-datasets."
+TBLPROPERTIES ("quality" = "bronze")
+AS
+--SELECT * FROM cloud_files("/databricks-datasets/retail-org/sales_stream/sales_stream.json/", "json", map("cloudFiles.inferColumnTypes", "true"))
+SELECT * FROM cloud_files("/databricks-datasets/retail-org/sales_orders/", "json", map("cloudFiles.inferColumnTypes", "true"))
+
+-- COMMAND ----------
+
+CREATE LIVE TABLE sales_orders_cleaned(
```

Comment:
Add additional comments here using '--' syntax to provide more clues as to what's going on. We want to have well commented code.